MC/bin/o2dpg_sim_workflow.py

```diff
@@ -908,6 +908,12 @@ def addQCPerTF(taskName, needs, readerCommand, configFilePath, objectsFile=''):
                    needs=[EMCRECOtask['name']],
                    readerCommand='o2-emcal-cell-reader-workflow --infile emccells.root',
                    configFilePath='json://${O2DPG_ROOT}/MC/config/QC/json/emc-digits-task.json')
+     ### FT0
+     if isActive('FT0'):
```

Comment:
Have you tested this? I am afraid that if you deactivate FT0, you will get an unfulfilled dependency for the QC finalization part for `RecPointsQC` (see the other file) in one-node setups.
If you use a flag to (de)activate FT0 QC here, it might be needed also in `o2dpg_qc_finalization_workflow`.